frontend/src/components/worksheets/items/TableItem/TableItem.js

```diff
@@ -148,20 +148,28 @@ class TableItem extends React.Component<{
                                   top: 0,
```

Comment:
Since the two conditions are identical except for paddingLeft, maybe consider doing something similar to https://github.com/codalab/codalab-worksheets/blob/master/frontend/src/components/worksheets/items/TableItem/BundleRow.js#L192-L197, so we won't need to update two places in the future